packages/react-storage/src/components/StorageBrowser/Views/Controls/__tests__/Table.spec.tsx

```diff
@@ -195,7 +195,7 @@ describe('LocationsViewTable', () => {
     await waitFor(() => {
       render(
         <Provider>
-          <LocationDetailViewTable />
```

Comment:
Not sure how this happened but this was testing the wrong table so the test was failing on this PR.

Reply:
I'll look into it, sorry that this happened

Reply:
No worries!